boa/code/version.py

```diff
@@ -0,0 +1,30 @@
+from boa.code.compiler import Neo2Compiler, Neo3Compiler
+from boa.code.converter import Neo2Converter, Neo3Converter
+
+
+class Version:
+    converter = None
+    compiler = None
+    __version = None
+
+    @staticmethod
+    def init(version):
+        if version is None:
+            raise Exception('Incompatible neo version: ' % version)
+
+        if version.startswith('2.'):
+            _init_neo2()
+        elif version.startswith('3.'):
+            _init_neo3()
+
+        Version.__version = version
+
+
+def _init_neo2():
```

Comment:
private methods on python starts with a double underscore, so it should be `__init_neo2()`